websites/S/Simplyaweeb/dist/metadata.json

```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "Simplyaweeb",
 	"description": {
-		"en": "Weebs only one stop to get updated with the latest anime content, meme and bang on the latest anime music. More features coming soon so stay tuned."
+		"en": "Weebs only one stop to get updated with the latest anime content, meme and bang on the latest anime music. More features coming soon so stay tuned.",
+		"fr": "Un seul arrêt pour être à jour avec les derniers animes, memes et la dernière ost. Plus de fonctionnalités sont à venir, donc restez à l'écoute."
 	},
 	"url": "simplyaweeb.to",
-	"version": "1.0.0",
+	"version": "1.1.0",
```

Comment:
```suggestion
	"version": "1.0.1",
```

Reply:
Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8